scss/_patterns_subnav.scss

```diff
@@ -0,0 +1,57 @@
+@import 'settings';
+
+@mixin vf-p-subnav {
+  .p-subnav {
+    position: relative;
+
+    > a {
+      padding-right: $sph-inner--x-large;
+    }
+
+    &::after {
```

Comment:
This styling should be above the link styling.

Reply:
done